nixos/modules/services/audio/navidrome.nix

```diff
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.navidrome;
+in {
+  options = {
+    services.navidrome = {
+      enable = mkEnableOption pkgs.navidrome.meta.description;
+      settings = mkOption rec {
+        type = types.attrs;
```

Comment:
Have you thought about using `(pkgs.fotmats.json {}).type` here instead? See https://github.com/NixOS/nixpkgs/blob/e7fe577d9803885d1191c6612b95c246cb605dde/nixos/doc/manual/development/settings-options.xml for a reference.

Reply:
didn't know this existed, thanks !